### ISSUE TITLE
Damage breaking at a certain point

### ISSUE THREAD

**Issue by GeezyWheezy** [Alexis-Grimm/Un-Limited#24]
I use this mod along with Endless Growth and my melee pawn's damage is set to 0 when they hit level 44, but they had upwards of 4k DPS before?

Is there any way this can be fixed or is it just a game limitation?

I know that EndlessGrowth just uncaps skills, while this mods unlimits the actual pawn's stats, so I'm assuming the problem is with Un-Limited?
Forgive me if I'm wrong, don't mean to waste your time

**Comment by Alexis-Grimm** [comment]
You would normally be right, but this mod doesn't actually touch damage stats so it shouldn't be this, I imagine this is a vanilla problem similar to the trade price improvement bug. Just to make sure, have you tested if this happens if you load the save without my mod?

**Comment by GeezyWheezy** [comment]
I haven't, but I think I found the problem?

I tested some more by unequipping my pawn's weapon and ended up seeing the issue lies with Tacticowl's Dual wield feature-- it boosts damage by alot for some reason (1 Sword is 148 DPS, with 2 it becomes 4348)

 So the 0 DPS counter on my melee stat was just a visual bug-- but now I'm questioning why this insanely boosted pawn's strikes were bouncing off a turtle? They're one shotting walls and other animals.
 
 So in the end this is probably a Combat Extended / Duel Wield issue. Sorry!
